package.json

```diff
@@ -23,6 +23,7 @@
     "i18n:pot": "grunt makepot",
     "i18n:check": "grunt checktextdomain",
     "i18n": "npm run -s i18n:js && npm run -s i18n:php && npm run -s i18n:check && npm run -s i18n:pot",
+    "phplint": "./vendor/bin/phpcs --standard=phpcs.xml.dist $(git ls-files | grep .php$)",
```

Comment:
Can we call this `lint:php` to follow the other commands (like `i18n`)? Maybe rename `lint` to `lint:js`, and then have `lint` run both commands?